src/main/java/it/pagopa/pn/deliverypush/action/AnalogWorkflowHandler.java

```diff
@@ -192,21 +195,32 @@ public void extChannelResponseHandler(ExtChannelAnalogSentResponseInt response)
 
         log.info("Analog workflow Ext channel response  - iun={} id={} with status={}", iun, recIndex, response.getStatusCode());
 
+        PnAuditLogBuilder auditLogBuilder = new PnAuditLogBuilder();
+        PnAuditLogEvent logEvent = auditLogBuilder
+                .before(PnAuditLogEventType.AUD_NT_CHECK, "Analog workflow Ext channel response iun={} id={} with status={}", iun, recIndex, response.getStatusCode())
+                .iun(iun)
+                .build();
+        logEvent.log();
+        
         if (status!= null) {
             switch (status) {
                 case OK:
+                    // AUD_NT_CHECK
                     // La notifica è stata consegnata correttamente da external channel il workflow può considerarsi concluso con successo
                     completionWorkFlow.completionAnalogWorkflow(notification, recIndex, legalFactsListEntryIds, response.getStatusDateTime(), sendPaperDetails.getPhysicalAddress(), EndWorkflowStatus.SUCCESS);
+                    logEvent.generateSuccess().log();
                     break;
                 case KO:
                     // External channel non è riuscito a effettuare la notificazione, si passa al prossimo step del workflow
                     int sentAttemptMade = sendPaperDetails.getSentAttemptMade() + 1;
                     analogWorkflowUtils.addAnalogFailureAttemptToTimeline(notification, sentAttemptMade, legalFactsListEntryIds, response.getDiscoveredAddress(), response.getDeliveryFailureCause()==null?null: List.of(response.getDeliveryFailureCause()),  sendPaperDetails);
                     nextWorkflowStep(notification, recIndex, sentAttemptMade);
+                    logEvent.generateFailure("External channel analogFailureAttempt proceding to next workflow step for iun={} id={}", iun, recIndex).log();
                     break;
             }
         } else {
             handleStatusProgress(response, iun, recIndex);
+            logEvent.generateFailure("Specified response={} is not final  - iun={} id={}", response.getStatusCode(), iun, recIndex).log();
```

Comment:
Siamo sicuri che è un failure?

Reply:
Si, in accordo con @Simone l'abbbiamo sostituito con una failure di AUT_NT_VALID

Reply:
Ok, sostitutire con AUT_NT_VALID, ma è una validazione di successo, non di fallimento